File: webservice/classes/area.php
<?php

Class Area{
    
    private $_id;
    private $_name;
    private $_color;
    
    function GetId() {
        return $this->_id;
    }

    function GetName() {
        return $this->_name;
    }

    function GetColor() {
        return $this->_color;
    }

    function SetId($_id) {
        $this->_id = $_id;
    }

    function SetName($_name) {
        $this->_name = $_name;
    }

    function SetColor($_color) {
        $this->_color = $_color;
    }

    function __construct($_id = null, $_name = null, $_color = null) {
        $this->_id = $_id;
        $this->_name = $_name;
        $this->_color = $_color;
    }

    
}

File: ios-client/Exametrics/Exametrics/Area.swift
//
//  Area.swift
//  Exametrics
//
//  Created by Vilaine Emilien on 18/05/2016.
//  Copyright © 2016 IMERIR. All rights reserved.
//

import Foundation
import Realm

class Area: RLMObject {
    
    // Champs
    private dynamic var _id    : String!
    private dynamic var _name  : String!
    private dynamic var _color : String!
    
    // Getter & Setter
    func getId() -> String{
        return _id
    }
    
    func setId(id: String) {
        _id = id
    }
    
    func getName() -> String{
        return _name
    }
    
    func setName(name: String) {
        _name = name
    }
    
    func getColor() -> String{
        return _color
    }
    
    func setColor(color: String) {
        _color = color
    }
    
    override init() {
        super.init()
    }
    // Constructeurs
    required init(id: String, name: String, color: String){
        
        _id = id
        _name = name
        _color = color
        super.init()
    }
}